CMakeLists.txt

```diff
@@ -24,41 +24,53 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 
+# New escape-sequence processing.
+cmake_minimum_required(VERSION 3.5)
+cmake_policy(SET CMP0053 NEW)
 
 # Don't check any language compiler. This project is for EditorConfig Core
 # testing only.
 project(editorconfig-core-test NONE)
 
 # Where this file lives
 set(tests_cmakelists_dir "${CMAKE_CURRENT_LIST_DIR}")
-#message(STATUS "Tests are in ${tests_cmakelists_dir}")
+message(STATUS "Tests are in ${tests_cmakelists_dir}")
 
 # Only when we are using editorconfig-core-test independently should we check
 # cmake version, set EDITORCONFIG_CMD as cache string, and enable_testing()
 # here.
 if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
-    cmake_minimum_required(VERSION 3.5)
     set(EDITORCONFIG_CMD "editorconfig" CACHE STRING "editorconfig command.")
     enable_testing()
+    message(STATUS "tests: Standalone testing enabled")
+endif()
+
+# Is the EDITORCONFIG_CMD of the form '*<target>'?
+string(REGEX MATCH "^=" is_target ${EDITORCONFIG_CMD})
```

Comment:
Rather than using a regex match, how about using an explicit option?

```cmake
option(USE_TARGET "EDITORCONFIG_CMD points to a target rather than an executable." OFF)
```

Reply:
It's fine with me.  I would prefer to use `set` rather than `option` so I don't have to worry about [CMP0077](https://cmake.org/cmake/help/v3.15/policy/CMP0077.html#policy:CMP0077), and for consistency with `EDITORCONFIG_CMD`.  I will push a commit to test that in a few minutes.